flink-libraries/flink-table/src/test/scala/org/apache/flink/table/api/scala/stream/table/GroupWindowAggregationsITCase.scala

```diff
@@ -179,6 +180,36 @@ class GroupWindowAggregationsITCase extends StreamingMultipleProgramsTestBase {
   }
 
   @Test
+  def testEventTimeTumblingWindowWithPojoData(): Unit = {
```

Comment:
The failure is unrelated to windows. Hence, I would not add the test to this class and remove the window. `TimeAttributesITCase` is a better place to validate that converting a DataStream of Pojos works with time attributes.